src/transport_ofi.c

```diff
@@ -559,8 +572,16 @@ static inline int atomic_limitations_check(void)
         ret = fi_atomicvalid(shmem_transport_ofi_epfd, SHM_DT_INT[i], SHM_BOPS[j],
                         &atomic_size);
         if(ret!=0 || atomic_size == 0) {
-           OFI_ERRMSG("ret=%d atomic_size=%d %d %d\n", ret, (int)atomic_size, i, j);
-	   return ret;
+            fprintf(stderr, "Atomic Support: data size %d not "\
+                    "supported with %s ", SHMEM_Dtsize[SHM_DT_INT[i]],
```

Comment:
Why Dtsize and not a string with the name of the datatype?  With only the size, one can't distinguish between long and double..